lib/dom.js

```diff
@@ -202,6 +202,7 @@ declare class Document extends Node {
     documentElement: HTMLElement;
     implementation: DOMImplementation;
     scripts: HTMLCollection;
+    styleSheets: HTMLCollection;
```

Comment:
looks like it's actually a [StyleSheetList](https://developer.mozilla.org/en-US/docs/Web/API/StyleSheetList). would you mind creating a type for that?